code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1401,14 +1401,20 @@
 	. = ..()
 	L.add_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
 	ADD_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
-	ADD_TRAIT(L, TRAIT_SIXTHSENSE, type)
+	ADD_TRAIT(L, TRAIT_RESISTHEAT, type)
 
 /datum/reagent/hexane/on_mob_end_metabolize(mob/living/L)
 	L.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
 	REMOVE_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
-	REMOVE_TRAIT(L, TRAIT_SIXTHSENSE, type)
+	REMOVE_TRAIT(L, TRAIT_RESISTHEAT, type)
 	return ..()
 
+/datum/reagent/hexane/on_mob_life(mob/living/L)
+	. = ..()
+	L.hallucination += 150
```

Comment:
um

you do know that this is a duration, not an intensity, right? each tick (2 seconds, I think) of this reagent beign in your system is going to apply 150 ticks' worth (5 minutes' worth if delta \_time hasn't changed things up) of hallucinations.

Reply:
ohhhhhh i thought it was some kind of intensity amount... what should be the right thing to do? just something like + 10 ?